src/pages/ManageUser/EachUser.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import MainContainer from '../../components/layout/MainContainer'
+import { ReactComponent as GoBack } from "../../assets/icons/goBack.svg";
```

Comment:
why not create an index file in assets and export all images from there?

Reply:
Done